src/components/Product-Page/index.jsx

```diff
@@ -0,0 +1,47 @@
+import { Typography } from "geeky-ui";
```

Comment:
The product page should have been placed under the pages/screens folder and not under components, we keep reusable components under the components folder.

Reply:
haha, right right 👍